src/envo/envo-edit.owl

```diff
@@ -40171,6 +39843,535 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3400002> "ar
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3510000> "wood dust"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3510000> <http://purl.obolibrary.org/obo/ENVO_00002008>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_3510001> (brake fluid)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://doi.org/10.1520/JTE14254") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Brake_fluid") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_3510001> "A hydraulic fluid which is 1) used in hydraulic brake and hydraulic clutch applications in vehicles to amplify braking force and 2) composed primarily of ethylene glycol, polyglycols, silicone fluids, and isobutyl alcohol.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_3510001> <http://orcid.org/0000-0002-7463-6306>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_3510001> "2020-08-03T17:32:09Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3510001> "brake fluid"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3510001> <http://purl.obolibrary.org/obo/ENVO_3510004>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_3510002> (automatic transmission fluid)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Automatic_transmission_fluid") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_3510002> "Fluid used for valve operation, brake band friction, and gear lubrication within an automatic vehicle transmission.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_3510002> <http://orcid.org/0000-0002-7463-6306>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_3510002> "2020-08-03T17:34:16Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_3510002> "ATF")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3510002> "automatic transmission fluid"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3510002> <http://purl.obolibrary.org/obo/ENVO_00003074>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_3510004> (hydraulic fluid)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Hydraulic_fluid") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_3510004> "A medium by which power is transferred in hydraulic machinery.")
```

Comment:
The definition here is a little off - this may be better when this class is positioned (asserted) in the right place. Something like "A liquid environmental material which ..." if you place it there